models/public/parking-vehicle-detection/model.yml

```
@@ -0,0 +1,40 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  The "cascade-RCNN" model is two-stage multibox Detection network intended to perform 
+  object detection. This model is implemented using mmdetection framework.
+
+  The model input is a blob that consists of a single image of 1x3x800x1333 in BGR
+  order. The BGR mean values need to be subtracted
+  as follows: [123.675, 116.28, 103.53] before passing the image blob into the network. 
+  Also BGR values need to be scaled by division on following coefficients: [58.395, 57.12, 57.375]  
+
+  The model output is a typical vector containing the tracked object data, as previously
+  described.
+task_type: detection
+files:
+  - name: parking-vehicle-detection.onnx
+    size: tbd
+    sha256: tbd
+    source: tbd
+model_optimizer_args:
+  - --input_shape=[1,3,800,1333]
+  - --input=image
+  - --mean_values=data[123.675, 116.28, 103.53]
+  - --scale_values=data[58.395, 57.12, 57.375]
+  - --input_model=$dl_dir/mobilenet-ssd.caffemodel
+  - --input_proto=$dl_dir/mobilenet-ssd.prototxt
```

@ygnn123 do not forget to update these lines, according your template above:
```suggestion
  - --input_model=$dl_dir/parking-vehicle-detection.onnx
```

---

If there are no problems with PyTorch to ONNX conversion, it's better to download PT model.